examples/convert_ieeg_to_bids.py

```diff
@@ -124,13 +127,33 @@
 print("The channel coordinates before writing into BIDS: ")
 pprint([x for x in zip(ch_names, pos)])
 
+###############################################################################
+# BIDS vs MNE-Python Coordinate Systems
+# -------------------------------------
+#
+# BIDS has many acceptable coordinate systems for iEEG, which can be viewed in
+# `appendix VIII`_ of the BIDS specification.
+# However, MNE-BIDS depends on MNE-Python and MNE-Python does not support all
+# these coordinate systems (yet).
+#
+# MNE-Python has a few tutorials on this topic:
+#
+#   - `background on FreeSurfer`_
+#   - `MNE-Python coordinate frames`_
+#
+# Currently, MNE-Python supports the ``mni_tal`` coordinate frame, which
+# corresponds to the ``fsaverage`` BIDS coordinate system. All other coordinate
+# frames in MNE-Python if written with :func:`mne_bids.write_raw_bids` are
+# written with coordinate system ``'Other'``. Note, then we suggest using
+# :func:`mne_bids.update_sidecar_json` to update the sidecar
+# ``*_coordsystem.json`` file to add additional information.
+
 ###############################################################################
```

Comment:
@adam2392 yes ... my fault: this bar of ################### needs to be removed

Reply:
and then the upper block of comments needs to be connected with the "step 2: formatting ...." block via a  # char

Reply:
see: https://github.com/mne-tools/mne-bids/pull/717#issuecomment-799448682